### DISCUSSION TITLE
Local setup of the QuML player in windows OS based systems

### DISCUSSION THREAD

**Discussion by HarishGangula** [Sunbird-inQuiry/Community#67] [Q&A]
While doing the local setup of the QuML player in windows systems there are issues with node packages installations, Need contributions or steps to solve this issue.

cc: @pallakartheekreddy  @Krishnaj20

**Comment by pallakartheekreddy** [comment]
@sajeshkayyath @PoojaJManjooran @panduankani @snehar-nd adding you in this thread as this issue we faced in bootcamp.

**Reply by panduankani** [reply]
In the bootcamp, we installed the recommended versions of Node and Angular CLI in order to run the QuML player. Following are the versions we installed and we were able to run it successfully in the windows OS:

Node - 12.11.1
Angular CLI - 9.1.15

**Comment by PoojaJManjooran** [comment]
> While doing the local setup of the QuML player in windows systems there are issues with node packages installations, Need contributions or steps to solve this issue.
> 
> cc: @pallakartheekreddy @Krishnaj20

@HarishGangula Thank you for raising this. We will look into this.
Attaching the jira ticket [here](https://project-sunbird.atlassian.net/browse/IQ-331)
We will update the ticket with plan and timelines to resolve the issue.

**Reply by sajeshkayyath** [reply]
@HarishGangula - We tested the steps against a fresh windows machine and the issue was uncovered when the dependent software were of higher versions than we currently support. The documentation has been revisited and updated. 

https://inquiry.sunbird.org/use/developer-installation/question-set-player/installation

cc @pallakartheekreddy / @rajnishdargan / @vaibhavbhuva / @krgauraw / @PoojaJManjooran / @kameshbhr
